packages/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -793,3 +794,73 @@ class WithDropdown extends React.Component {
   }
 }
 ```
+
+### With help
+
+```js
+import React from 'react';
+import { Modal, Button, Popover } from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/js/icons/help-icon';
+
+class HelpModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false
+    };
+    this.handleModalToggle = () => {
+      this.setState(({ isModalOpen }) => ({
+        isModalOpen: !isModalOpen
+      }));
+    };
+  }
+
+  render() {
+    const { isModalOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Button variant="primary" onClick={this.handleModalToggle}>
+          Show Modal
+        </Button>
+        <Modal
+          title="Simple modal header"
+          help={
+            <Popover
+              headerContent={<div>Help Popover</div>}
+              bodyContent={
+                <div>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla
+                  turpis.
+                  <Button onClick={() => console.log('click')}>click</Button>
```

Comment:
I don't know that we need this button in the example.  It looks a little off to me.  @mcarrano can you take a look at the popover content.  I am not sure that the `click` button would follow design recommendations.

Reply:
I will remove it.

Also, there is no actual `pf-c-modal-box__header-help` class, it doesn't do anything. It is a placeholder class to indicate the div.